apps/ledger-live-mobile/src/newArch/features/Accounts/components/AccountQuickActionsDrawer/useAccountQuickActionDrawerViewModel.ts

```diff
@@ -27,14 +28,14 @@ type ActionItem = {
 
 export default function useAccountQuickActionDrawerViewModel({
   currency,
-  accounts,
+  account,
 }: {
   currency: CryptoOrTokenCurrency;
-  accounts: Account[] | TokenAccount[];
+  account: Account | TokenAccount;
 }) {
   const {
     quickActionsList: { RECEIVE, BUY },
-  } = useQuickActions({ currency, accounts });
+  } = useQuickActions({ currency, accounts: [account ?? {}] });
```

Comment:
Do we need this `?? {}` ? Looking at the props type, seems like `account` should always be defined